libs/core/src/lib/carousel/carousel-item/carousel-item.component.ts

```diff
@@ -18,7 +20,10 @@ let carouselItemUniqueId = 0;
     changeDetection: ChangeDetectionStrategy.OnPush,
     encapsulation: ViewEncapsulation.None
 })
-export class CarouselItemComponent implements CarouselItemInterface {
+export class CarouselItemComponent implements CarouselItemInterface, OnInit {
+    /** add #imgInfo to <img> to get attr.alt*/
+    @ContentChild('imgInfo', { static: true }) imgInfoRef: ElementRef;
```

Comment:
I'm not sure we need this at all. `alt` on the image should be enough, it's being read for me by VoiceOver on Mac.

Reply:
it was read, but title was not shown
Below implementation is required in order for fd-carousel-item to have a title:
    ngOnInit(): void {
        if (!!this.imgInfoRef) {
            this.title = this.imgInfoRef.nativeElement.getAttribute('alt');
        }
    }

Reply:
Okay, so if the carousel-item needs a title let's add it as a separate input. Because, usually `alt` is enough and yea, it's different from `title` because there is no tooltip for `alt` shown but it's intended. If we want title - let's add it ;)